### PR TITLE
[web] update TODO format to match flutter repo and fix names, more code cleanup

### PR DESCRIPTION
**Pull request by ferhatb**
Related issue:
flutter/flutter#86891

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on
writing and running engine tests.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

### CODE REVIEW

File: lib/web_ui/lib/src/engine/canvaskit/font_fallbacks.dart [hunk 1]
```diff
@@ -56,6 +56,7 @@ class FontFallbackData {
       //                resetting the tree would be a matter of reconstructing
       //                the new resolved tree.
       font.reset();
+      //ignore: prefer_foreach
```
Comment (mdebbar) [inline lib/web_ui/lib/src/engine/canvaskit/font_fallbacks.dart:59]: nit:
```suggestion
      // ignore: prefer_foreach
```

File: lib/web_ui/lib/src/engine/text/paragraph.dart [hunk 2]
```diff
@@ -85,10 +85,9 @@ class EngineLineMetrics implements ui.LineMetrics {
     required this.descent,
     // Didn't use `this.boxes` because we want it to be non-null in this
     // constructor.
```
Comment (mdebbar) [inline lib/web_ui/lib/src/engine/text/paragraph.dart:87]: This comment isn't valid anymore.
```suggestion
```

**Review by mdebbar — APPROVED** [review]

File: lib/web_ui/lib/src/engine/canvaskit/font_fallbacks.dart (hunk 1 above)
Comment (ferhatb) [reply lib/web_ui/lib/src/engine/canvaskit/font_fallbacks.dart]: Done.

File: lib/web_ui/lib/src/engine/text/paragraph.dart (hunk 2 above)
Comment (ferhatb) [reply lib/web_ui/lib/src/engine/text/paragraph.dart]: Done.

**Comment by ferhatb** [comment]
merging on infra fail.

**Pull request merged**